packages/askui-nodejs/src/execution/ui-controller-client-interface.ts

```diff
@@ -56,11 +57,13 @@ export interface ClientArgs {
   readonly reporter?: Reporter | Reporter[] | undefined
   readonly context?: ContextArgs | undefined
   readonly inferenceServerApiVersion?: string
+  readonly retryStrategy?: RetryStrategy
```

Comment:
Let's document it in the jsdoc, also documenting the default value.

Reply:
I will work with @JohannesDienst on this

Reply:
In this PR as this is part of code, right?

FYI: Just for clarification. I don't refer to docs.askui.com but the jsdoc above the interface definition.

Reply:
Done.